README.rst

```diff
@@ -41,877 +43,10 @@ as a requirements management tool for e.g. ISO26262 projects.
-*attribute_*-keys are mandatory to use the *checklist-item* directive. The other configuration variables are only used
-for querying GitLab/GitHub.
-
-.. code-block:: python
-
-    traceability_checklist = {
-        'attribute_name': 'your_attribute_name',
-        'attribute_to_str': 'your_attribute_to_string'),
-        'attribute_values': 'your_attribute_values',  # two values, comma-separated
-        'private_token': 'your_private_token',  # optional, depending on accessibility
-        'api_host_name': 'https://api.github.com' or 'https://gitlab.example.com/api/v4',
-        'project_id': 'the_owner/your_repo' or 'your_project_id',
-        'merge_request_id': 'your_merge_request_id(s)'),  # comma-separated if more than one
-        'checklist_item_regex': 'your_item_id_regex',  # optional, the default is r"\S+"
-    }
-
-If the *checklist_item_regex* is configured, a warning is reported for each item ID that matches it and is not defined
-with the *checklist-item* directive.
-
-Configuration via .env file
-```````````````````````````
-In our *conf.py* the variables are looked for in the environment first, e.g. in a ``.env`` file (by using the Python
-*decouple* module).
-
-.. code-block:: bash
-
-    # copy example .env to your .env
-    cp doc/.env.example .env
-
-    # add env variables by adjusting the template values in .env
-
-Common variables
-````````````````
-- *ATTRIBUTE_NAME* is the identifier of the attribute to be added, e.g. *checked*.
-- *ATTRIBUTE_TO_STRING* is the string representation (as to be rendered in html) of the attribute name, e.g. *Answer*.
-- *ATTRIBUTE_VALUES* are two comma-separated attribute values, e.g. *yes,no*. The first value is used when the checkbox is checked and the second value when unchecked.
-
-Query-specific variables
-````````````````````````
-GitLab
-''''''
-- *PRIVATE_TOKEN* is your personal access token that has API access.
-- *API_HOST_NAME* is the host name of the API, e.g. *https://gitlab.example.com/api/v4*
-- *PROJECT_ID* is the ID of the project.
-- *MERGE_REQUEST_ID* is the internal ID of the merge request.
-
-GitHub
-''''''
-- *PRIVATE_TOKEN* is not needed for public repositories. Otherwise, it must be a `personal access token`_ with the access to the targeted scope.
-- *API_HOST_NAME* is the host name of the GitHub REST API v3: *https://api.github.com*
-- *PROJECT_ID* defines the repository by specifying *owner* and *repo* separated by a forward slash, e.g. *melexis/sphinx-traceability-extension*.
-- *MERGE_REQUEST_ID* is the pull request number.
-
-.. _`personal access token`: https://github.blog/2013-05-16-personal-api-tokens/
-
-.. _traceability_process:
-
--------
-Process
--------
+.. toctree::
+    :maxdepth: 2
 
-The Melexis.SWCC process has a Guideline for documenting in reStructuredText (RST). It holds guidelines for using
-the traceability plugin with naming conventions, templates, etc.
```

Comment:
You lost this paragraph. I suggest it stays on front page on main readme. It is a good indication for us. (Maybe spell out swcc as well)

Reply:
Added by 3f162304892bf4de329a821752b2ab5b53fdf361